lib/prisma.ts

```diff
@@ -18,5 +18,6 @@ const prisma = globalThis.prismaGlobal ?? prismaClientSingleton();
 
 export default prisma;
 
-if (process.env.NODE_ENV !== "production")
+if (process.env.NODE_ENV !== "production") {
```

Comment:
This is now enforced, because [lessOpinionated](https://github.com/antfu/eslint-config/blob/8942cc0c18e181998ab54bc72c3dc94f19e3e777/src/configs/stylistic.ts#L54) reverts us to the eslint 'curly' rule being set to 'error' in 'all' cases: https://eslint.org/docs/latest/rules/curly